iree/samples/static_library/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# if(${IREE_BUILD_STATIC_LIBRARY_SAMPLES} AND NOT ${IREE_BUILD_SAMPLES})
+#   return()
+# endif()
+
+if(NOT ${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT}
+  OR NOT ${IREE_HAL_DRIVER_DYLIB}
+  OR NOT ${IREE_BUILD_COMPILER}
+  OR NOT ${IREE_BUILD_SAMPLES})
```

Comment:
remove, `samples/` isn't added at the root if this isn't set:

https://github.com/google/iree/blob/4d3b33a0f3e9a5d9266463efbafb6cb2be9e3251/CMakeLists.txt#L475-L477